internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -102,9 +102,10 @@ func (e *EnvStackConfig) Template() (string, error) {
 		ArtifactBucketARN:      e.in.ArtifactBucketARN,
 		ArtifactBucketKeyARN:   e.in.ArtifactBucketKeyARN,
 
-		ImportCertARNs: e.importCertARNs(),
-		VPCConfig:      e.vpcConfig(),
-		Telemetry:      e.telemetryConfig(),
+		ImportCertARNs:     e.importCertARNs(),
+		VPCConfig:          e.vpcConfig(),
+		InternalALBSubnets: e.internalALBSubnets(),
```

Comment:
nit: I wonder if this would be easier to read in the template if we renamed it to:
```suggestion
		CustomInternalALBSubnets: e.internalALBSubnets(),
```

Reply:
🤔 Because `{{- if .InternalALBSubnets}}` doesn't really distinguish the specified target subnets from regular subnets the internal ALB is being placed on, right? Yeah, that makes sense. Changing.